### ISSUE TITLE
Fengtien Republic Missing Event

### ISSUE THREAD

**Issue by treatss** [Kaiserreich/Kaiserreich-DH#36]
An event for the Fengtien Republic deciding to abandon Japanese overlordship and join the Qing is mentioned in 4 different events but doesn't exist - event id 627066. A comment claims the event is in Fengtien Republic.txt but it isn't there, it might be that the event was removed but never replaced.

Event 627066 is triggered by:
Japan_Korea.txt id = 340087 for Fengtien after successful Korean revolt and choosing to "reunite China under Pu Yi"
Taiwan.txt id = 156013 for Fengtien after Korea and Taiwan ally against Japan and choosing to "reunite China under Pu Yi"
Taiwan.txt id = 156018 for Fengtien after Japan tries restoring alliance with rebelling colonies/puppets by granting them independence, and Fengtien chooses to "reunite China under Pu Yi"
Qing Foreign Policy.txt id = 148103 for (Japanese puppeted) Fengtien after Qing targets Fengtien, Fengtien agrees to cooperate, Qing declares war on Japan, and Fengtien leaves the Japanese alliance and stops being a puppet

**Comment by eberronguy** [comment]
It appears you are correct, as I cannot find that event in any place where it would logically be. What I am unsure of is what the event would be about, since event 144012 in the Fengtian file does the same thing. Do you think those should be corrected to reference that event?

**Comment by eberronguy** [comment]
After testing in game the event is indeed missing, and event 148103 doesn't even give you an option that is supposed to lead to the event like the others do.

**Comment by treatss** [comment]
For event 148103 were you missing the flag MAN_yes for the option that leads to the event? I discovered this whole issue because it seemed the AI in my game took it, which left Fengtien independent and outside the Japanese alliance but together in war with them against Qing. But then again event 148103 could be replaced and removed entirely, since it is only triggered by an event that requires flag MAN_yes where Fengtien is supposed to agree to join the Qing.

By the way, Event 144012 has an option that causes Qing to inherit MAN, and the same option triggers event 144015, that can lead Japan to cede provinces to MAN, a few other events have this as well, seems like a separate bug. 

I went and created a fork of my own to work with, let me know what you think
